lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/util/EvidenceUtils.java

```diff
@@ -16,6 +16,11 @@ public static int getVerificationScoreForPass(EvidenceRequest evidenceRequest) {
             return VC_PASS_EVIDENCE_SCORE;
         }
         int verificationScore = evidenceRequest.getVerificationScore();
+        if (verificationScore > VC_PASS_EVIDENCE_SCORE) {
```

Comment:
The if-condition means we don't need a ternary in the return and you can just return the verification score. You might want to do a less-than-zero check to be safe as you won't capture a score of 0, -1, -2 ... 

Though, what's the reason for adding this block? If in the future we have a requirement of a score of 3 then this logic will need to be removed or re-written. A better solution would be to figure what the highest score we currently support is and do the conditional on that but that can be a bit complicated for this PR/ticket.

Reply:
I'm okay to leave this as you have done it though